shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitParameters.java

```diff
@@ -51,13 +65,36 @@ public static ShortCircuitParameters load(PlatformConfig platformConfig) {
                         .setWithFeederResult(config.getBooleanProperty("with-feeder-result", DEFAULT_WITH_FEEDER_RESULT))
                         .setStudyType(config.getEnumProperty("study-type", StudyType.class, DEFAULT_STUDY_TYPE))
                         .setWithFortescueResult(config.getBooleanProperty("with-fortescue-result", DEFAULT_WITH_FORTESCUE_RESULT))
-                        .setMinVoltageDropProportionalThreshold(config.getDoubleProperty("min-voltage-drop-proportional-threshold", DEFAULT_MIN_VOLTAGE_DROP_PROPORTIONAL_THRESHOLD)));
+                        .setMinVoltageDropProportionalThreshold(config.getDoubleProperty("min-voltage-drop-proportional-threshold", DEFAULT_MIN_VOLTAGE_DROP_PROPORTIONAL_THRESHOLD))
+                        .setSubTransientCoefficient(config.getDoubleProperty("sub-transient-coefficient", DEFAULT_SUB_TRANSIENT_COEFFICIENT))
+                        .setWithLoads(config.getBooleanProperty("with-loads", DEFAULT_WITH_LOADS))
+                        .setWithShuntCompensators(config.getBooleanProperty("with-shunt-compensators", DEFAULT_WITH_SHUNT_COMPENSATORS))
+                        .setWithVSCConverterStations(config.getBooleanProperty("with-vsc-converter-stations", DEFAULT_WITH_VSC_CONVERTER_STATIONS))
+                        .setWithNeutralPosition(config.getBooleanProperty("with-neutral-position", DEFAULT_WITH_NEUTRAL_POSITION))
+                        .setInitialVoltageProfile(config.getEnumProperty("initial-voltage-profile", InitialVoltageProfile.class, DEFAULT_INITIAL_VOLTAGE_PROFILE))
+                        .setConfiguredInitialVoltageProfileCoefficients(getCoefficientsFromConfig(config)));
+
+        if (parameters.initialVoltageProfile == InitialVoltageProfile.CONFIGURED && parameters.configuredInitialVoltageProfileCoefficients.isEmpty()) {
+            throw new PowsyblException("Configured initial voltage profile but coefficients are missing.");
+        }
 
         parameters.readExtensions(platformConfig);
 
         return parameters;
     }
 
+    private static List<ConfiguredInitialVoltageProfileCoefficient> getCoefficientsFromConfig(ModuleConfig config) {
+        List<ConfiguredInitialVoltageProfileCoefficient> coefficients = new ArrayList<>();
+        config.getOptionalStringListProperty("configured-initial-voltage-range-coefficients").ifPresent(voltageCoefficients ->
+                voltageCoefficients.forEach(coefficient -> {
+                    String[] voltageCoefficientArray = coefficient.split(" -> ");
+                    String[] voltages = voltageCoefficientArray[0].split("-");
+                    coefficients.add(new ConfiguredInitialVoltageProfileCoefficient(Integer.parseInt(voltages[0]), Integer.parseInt(voltages[1]), Double.parseDouble(voltageCoefficientArray[1])));
```

Comment:
Since the constructor of `ConfiguredInitialVoltageProfileCoefficient` takes doubles for its 2 first parameters, shouldn't we accept doubles in the yml file?

If this is a matter of readability of the file, we may allow additional spaces around the values (to trim when reading the file):
```groovy
- 55.9-63.2 -> 14
- 220.3-360.3 -> 12.9
```
is not very readable, but the following is:
```groovy
-  55.9 -  63.2 -> 14
- 220.3 - 360.3 -> 12.9
```

Reply:
You are right, I changed it so that spaces do not impact reading the file